### ISSUE TITLE
aria-label missing on progress bars

### ISSUE THREAD

**Issue by matsbryntse** [bryntum/support#7634]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=26566&p=132387#p132387)

Hi there,

We use Bryntum alongside lightning web components in Salesforce.

After running a lighthouse report on google chrome, we found that there are missing aria-labels for our progress bars on the scheduler pro. The screenshot below displays the error we receive and the auto-generated properties. As you can see, the aria-label property is missing in the div.

The progress-bar is a column created in our scheduler config. Is there a property that we can add to the column to enter the value for the aria-label or should this be autogenerated by Bryntum?
Please let us know if this is a bug or if there is something that we need to do on our end.
![image (6).png](https://forum.bryntum.com/download/file.php?id=24494)
![image (5).png](https://forum.bryntum.com/download/file.php?id=24493)
![image (4).png](https://forum.bryntum.com/download/file.php?id=24492)
